requirements.dev.txt

```diff
@@ -1,151 +1,159 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
 #    pip-compile --allow-unsafe --output-file=requirements.dev.txt requirements.dev.in
 #
-attrs==21.4.0
-    # via
-    #   flake8-implicit-str-concat
-    #   hypothesis
-    #   pytest
-black==22.3.0
+attrs==24.3.0
+    # via hypothesis
+black==24.8.0
     # via -r requirements.dev.in
+build==1.2.2.post1
+    # via pip-tools
 certifi==2020.11.8
     # via
-    #   -c requirements.prod.txt
+    #   -c /home/wavy/bennett/job-runner/requirements.prod.txt
```

Comment:
this is very mildly irritating but not really worth the manual fixup

Reply:
I think this is a consequence of updating the pip version, and AFAICT, there's not much we can do about it, other than maybe post-processing the .txt files to remove the path?